namron_dimmer.js

```diff
@@ -0,0 +1,338 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "Z-Wave dimmer 2 400W",
+	"description": "In-Wall Dimmer Module",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xd00c"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "Saving load state",
+			"description": "Saving load state before power failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+			"options": [
+			  {
+					 "label": "Shut off load",
+					 "value": 0
+			  },
+			  {
+					 "label": "Turn on load",
+					 "value": 1
+			  },
+			  {
+					 "label": "Save load state before power failure"
+					 "value": 2
+			  }
+		   ]	 
+		},
+		"3": {
+			"label": "Enable/disable to send report",
+			"description": " Enable/disable to send the basic report to the Lifeline when the load state changed（When value set as 1, repower on the dimmer, it will send Basic report automatically）",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+			"options": [
+			  {
+					 "label": "Disable to send Basic report",
+					 "value": 0
+			  },
+			  {
+					 "label": "Enable to send Basic report",
+					 "value": 1
+			  }
+		   ]
+		},
+		"4": {
+			"label": "Default fade time (in seconds)",
+			"description": "Default fade time (unit is second, this value has the same function as Duration of Multilevel) Valid value: 0~127",
```

Comment:
Quote from the style guide:
> unnecessary descriptions that merely restate the label must be removed. Additionally, information like units or available ranges should be removed as that information is provided to UIs through other properties.

Reply:
Ok, thanks for guiding me.
I've edited the file now. Hopefully it looks better now.